### DISCUSSION TITLE
How to move OSM and data from current samsung phone to a new one?

### DISCUSSION THREAD

**Discussion by yvesbodson** [osmandapp/OsmAnd#11767] [Q&A]
I have purchased an OSM license and I will need to move it from my current phone to a new samsung phone.
I wonder what is the procedure to move the licences and the data to the new phone?
I did check on topics such as moving to external cards or changing path but that does not really provide for a workflow allowing to move the data and the licence from one phone to a new one..
Thanks for any help on this.

**Comment by dmpr0** [comment]
Hello!
All your purchases in Google Play are linked to your Google account. On the new device, you need to add the same Google account, purchased OsmAnd version will be available automatically.

To move data from one OsmAnd to another you can use "Export", you can find it in OsmAnd – Menu – Settings – Export, you can select all or only needed data.

**Reply by Bergkraxler** [reply]
Export / yes, but I did not find guidance how to use it in the help menu of https://osmand.net/help-online
I have entered a support questions here: https://github.com/osmandapp/OsmAnd/issues/12009 
P.S. Why is this thread registered as unanswered?

**Comment by yvesbodson** [comment]
Ok thank you this is a relieve..  
Regards

    On Wednesday, May 26, 2021, 10:15:18 AM GMT+2, Dmitriy Prodchenko ***@***.***> wrote:  
 
 


Hello!
All your purchases in Google Play are linked to your Google account. On the new device, you need to add the same Google account, purchased OsmAnd version will be available automatically.

To move data from one OsmAnd to another you can use "Export", you can find it in OsmAnd – Menu – Settings – Export, you can select all or only needed data.

**Comment by yvesbodson** [comment]
Thanks for your help..

    On Saturday, June 26, 2021, 6:41:03 PM GMT+2, Bergkraxler ***@***.***> wrote:  
 
 


Export / yes, but I did not find guidance how to use it in the help menu of https://osmand.net/help-online
I have entered a support questions here: #12009
P.S. Why is this thread registered as unanswered?
